fuzzy_logic/fuzzy_operations.py

```diff
@@ -17,20 +17,24 @@ class FuzzySet:
     """
     A class for representing and manipulating triangular fuzzy sets.
     Attributes:
-        name: The name or label of the fuzzy set.
-        left_boundary: The left boundary of the fuzzy set.
-        peak: The peak (central) value of the fuzzy set.
-        right_boundary: The right boundary of the fuzzy set.
+    name: The name or label of the fuzzy set.
+    left_boundary: The left boundary of the fuzzy set.
+    peak: The peak (central) value of the fuzzy set.
+    right_boundary: The right boundary of the fuzzy set.
```

Comment:
Revert this change...  Indentation is customary.

Reply:
ok